lib/normalizer.c

```diff
@@ -1707,7 +1707,7 @@ grn_nfkc_normalize_unify_katakana_di_sound(grn_ctx *ctx,
 }
 
 static const unsigned char *
-grn_nfkc_normalize_unify_katakana_g_sounds(grn_ctx *ctx,
+grn_nfkc_normalize_unify_katakana_gu_small_sounds(grn_ctx *ctx,
                                            const unsigned char *start,
```

Comment:
Indent...

Reply:
Sorry...

Reply:
We need our formatter configuration and a lint CI job...

Reply:
Fixed.